packages/odie-client/src/data/use-get-support-interactions.ts

```diff
@@ -34,5 +34,6 @@ export const useGetSupportInteractions = (
 			return response;
 		},
 		enabled,
+		staleTime: 1000 * 10, // 10 seconds
```

Comment:
I wasn't sure which duration to choose, happy to take any suggestion 🙂.

Reply:
I would be tempted to do so but I'm wondering why those queries were not cached in the first place - is it because the author forgot about that, or deliberately chose to avoid it to make sure we present the latest reply from Zendesk as soon as it is posted from there?

Reply:
I believe it was due to forget about that? But yeah... SInce it was not there at least 10 seconds is better than nothing